docs/guide/install-and-run.asciidoc

```diff
@@ -122,48 +150,8 @@ Make sure you choose a good service name before you get started.
 The service name can only contain alphanumeric characters,
 spaces, underscores, and dashes (must match `^[a-zA-Z0-9 _-]+$`).
 
-[[nodejs-agent]]
-[float]
-==== Install the Node.js agent
-
-The Node.js agent automatically instruments Express,
-hapi,
-Koa,
-and Restify out of the box.
-See the {apm-node-ref}/index.html[APM Node.js Agent documentation] for more information.
-
-[[python-agent]]
-[float]
-==== Install the Python agent
-
-The Python agent automatically instruments Django and Flask out of the box.
-See the {apm-py-ref}/index.html[APM Python Agent documentation] for more information.
-
-[[ruby-agent]]
-[float]
-==== Install the Ruby agent
-
-The Ruby agent automatically instruments Rails out of the box.
-See the {apm-ruby-ref}/index.html[APM Ruby Agent documentation] for more information.
-
-[[java-agent]]
-[float]
-==== Install the Java Agent
-
-The Java agent automatically instruments Servlet API, Spring MVC, and Spring Boot out of the box.
-See the {apm-java-ref}/index.html[APM Java Agent documentation] for more information.
-
-[[go-agent]]
-[float]
-==== Install the Go Agent
-
-The Go agent automatically instruments Gorilla and Gin,
-as well as support for Go's built-in net/http and database/sql drivers.
-See the {apm-go-ref}/index.html[APM Go Agent documentation] for more information.
-
-[[rum-agent]]
 [float]
-==== Install the RUM JavaScript agent
+=== Next steps
 
-Real User Monitoring (RUM) captures user interactions with clients such as web browsers.
-See the {apm-rum-ref}/index.html[APM RUM JavaScript Agent documentation] for more information.
+Just like that, you're up and running with Elastic APM!
```

Comment:
hmm the cheerful language is nice if you are 100% convinced that getting APM running is a piece of cake, but might backfire if users run into problems... (component non reachable, etc). up to you ofc, this is a matter of taste ;)